test/cvm-win.json

```diff
@@ -13,7 +13,7 @@
       "cluster_name": "QWERTY",
       "vm_size": "Standard_DC1as_v4",
       "security_profile": {
```

Comment:
if both secureboot and vtpm are true by default, we don't need this section

Reply:
Resolved this in the new iteration.